src/ui/ExportToExcel/ExportToExcel.ts

```diff
@@ -13,6 +13,14 @@ import 'styling/_ExportToExcel';
 import { SVGIcons } from '../../utils/SVGIcons';
 import { SearchInterface } from '../SearchInterface/SearchInterface';
 import { get } from '../Base/RegisteredNamedMethods';
+import * as moment from 'moment';
+import { IQuery } from '../../rest/Query';
+
+let createAnchor = () => document.createElement('a');
+
+export function setCreateAnchor(fn: () => HTMLAnchorElement) {
+  createAnchor = fn;
+}
```

Comment:
Instead of using a `createAnchor` callback for unit tests to overwrite, I think it may be cleaner to create a separate utility class or function with a static `download` method.

Reply:
Hmm, if we do this, it would be fine for testing `download`, but wouldn't we still have difficulty testing the internal implementation `downloadExcelFile`?

Reply:
hey Benjamin, I will keep the current solution for this PR, but can revisit it in a future one,